### PR TITLE
Test Playbook For 'Possible External RDP Brute-Force - Set Verdict'

### PR DESCRIPTION
**Pull request by TalNos**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: Test Playbook For 'Possible External RDP Brute-Force - Set Verdict'

## Description
New test playbook for the 'Possible External RDP Brute-Force - Set Verdict' XSIAM playbook.

### CODE REVIEW

File: Packs/Core/TestPlaybooks/Test Playbook - Possible External RDP Brute-Force - Set Verdict.yml
Comment (ssokolovich) [inline Packs/Core/TestPlaybooks/Test Playbook - Possible External RDP Brute-Force - Set Verdict.yml:1]: the file name is not in our regular convention

File: Packs/Core/TestPlaybooks/Test Playbook - Possible External RDP Brute-Force - Set Verdict.yml [hunk 1]
```diff
@@ -0,0 +1,1456 @@
...
+    nexttasks:
+      '#none#':
+      - "309"
+    scriptarguments:
+      keys:
+        simple: risk_level,id
+      parent:
+        simple: Core.RiskyHost
+      values:
+        simple: MED,1111
```
Comment (ssokolovich) [inline Packs/Core/TestPlaybooks/Test Playbook - Possible External RDP Brute-Force - Set Verdict.yml:554]: Change this to be different from the endpoint
```suggestion
        simple: HIGH,2222
```

File: Packs/Core/TestPlaybooks/Test Playbook - Possible External RDP Brute-Force - Set Verdict.yml [hunk 2]
```diff
@@ -0,0 +1,1456 @@
...
+      "yes":
+      - "305"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isNotEmpty
+          left:
+            value:
+              simple: alert.alertsearchresults
```
Comment (ssokolovich) [inline Packs/Core/TestPlaybooks/Test Playbook - Possible External RDP Brute-Force - Set Verdict.yml:1126]: Should it be `xdralertsearchresilts`?
image
image

File: Packs/Core/TestPlaybooks/Test Playbook - Possible External RDP Brute-Force - Set Verdict.yml (hunk 1 above)
Comment (TalNos) [reply Packs/Core/TestPlaybooks/Test Playbook - Possible External RDP Brute-Force - Set Verdict.yml:554]: changed

File: Packs/Core/TestPlaybooks/Test Playbook - Possible External RDP Brute-Force - Set Verdict.yml
Comment (TalNos) [reply Packs/Core/TestPlaybooks/Test Playbook - Possible External RDP Brute-Force - Set Verdict.yml:1]: fixed

**Review by ssokolovich — APPROVED** [review]

**Pull request closed without merging**